cloud/logging.rst

```diff
@@ -26,34 +26,108 @@ The Citus Cloud dashboard automatically shows you the most recent 100 log lines
++------------------+--------------------------------------------------------------------------------------------------+
+| TLS              | No                                                                                               |
++------------------+--------------------------------------------------------------------------------------------------+
+| Protocol         | BSD Syslog over TCP                                                                              |
++------------------+--------------------------------------------------------------------------------------------------+
+| Message Template | <${PRI}>1 ${ISODATE} ${HOST} ${PROGRAM} ${PID} ${MSGID} [<token>@41058 tag=\\"CITUS\\" ] $MSG\\n |
++------------------+--------------------------------------------------------------------------------------------------+
+
+**Sumologic**
+
++------------------+---------------------------------------------------------------------------------+
+| Hostname         | syslog.collection.us2.sumologic.com                                             |
++------------------+---------------------------------------------------------------------------------+
+| Port             | 6514                                                                            |
++------------------+---------------------------------------------------------------------------------+
+| TLS              | Yes                                                                             |
++------------------+---------------------------------------------------------------------------------+
+| Protocol         | IETF Syslog                                                                     |
++------------------+---------------------------------------------------------------------------------+
+| Message Template | <${PRI}>1 ${ISODATE} ${HOST} ${PROGRAM} ${PID} ${MSGID} [<token>@41123] $MSG\\n |
++------------------+---------------------------------------------------------------------------------+
+
+**Logentries**
+
++------------------+--------------------------------+
+| Hostname         | data.logentries.com            |
++------------------+--------------------------------+
+| Port             | 80                             |
++------------------+--------------------------------+
+| TLS              | No                             |
++------------------+--------------------------------+
+| Protocol         | IETF Syslog                    |
++------------------+--------------------------------+
+| Message Template | <token> $ISODATE $HOST $MSG\\n |
++------------------+--------------------------------+
+
+**Other**
+
+We support other providers that can receive syslog via the BSD or IETF protocols. Internally Citus Cloud uses syslog-ng, so check your providers configuration documentation for syslog-ng settings.
+
+Please reach out if you encounter any issues.
+
+External Notifications
```

Comment:
The Logging title seems to imply server/db logs, whereas these are higher-level Citus Cloud notifications. Should be maybe rename this to more like "monitoring" as a top-level section? Don't know, just wondering.